app/controllers/concerns/recommend_webauthn_platform_concern.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RecommendWebauthnPlatformConcern
+  def recommend_webauthn_platform_for_sms_user?
+    # Only consider for A/B test if:
+    # 1. Option would be offered for setup
+    # 2. User is viewing content in English
+    # 3. Other recommendations have not already been offered (e.g. PIV/CAC for federal emails)
+    # 4. User selected to setup phone or authenticated with phone
+    # 5. User has not already set up a platform authenticator
+    return false if !device_supports_platform_authenticator_setup?
+    return false if I18n.locale != :en
+    return false if current_user.webauthn_platform_recommended_dismissed_at?
+    return false if !user_set_up_or_authenticated_with_phone?
+    return false if current_user.webauthn_configurations.platform_authenticators.present?
+    ab_test_bucket(:RECOMMEND_WEBAUTHN_PLATFORM_FOR_SMS_USER) == :recommend
```

Comment:
`ab_test_bucket` returns either one of the buckets they've been assigned to if they're in the X% group, otherwise it returns `:default`. So this checks that they're part of the group.